src/uiveri5.js

```diff
@@ -581,9 +596,32 @@ function run(config) {
       return driverActions.mouseMove(bodyElement, {x:-1, y:-1}).perform();
     }
 
+    function _moduleExec(modules,method,args) {
+      return Promise.all(
+        modules.map(function(module) {
+          return module[method].apply(null,args)
```

Comment:
what if the plugin doesn't define all hooks?

Reply:
good catch, will fix this